src/main/java/pivotaltracker/PageTransporter.java

```diff
@@ -48,10 +48,34 @@ public static void navigatePage(final String keyUrl) {
         goToURL(baseUrl.concat(keyUrl));
     }
 
+    /**
+     * Navigates on page with id.
+     *
+     * @param keyUrl - Url key.
+     * @param id     - Project id.
+     */
+    public static void navigatePageThroughId(final String keyUrl, final int id) {
+        String baseUrl = ConfigFileReader.getInstance().getProperties().get("baseUrl");
+        goToURL(baseUrl.concat(keyUrl).concat("/").concat(String.valueOf(id)));
+    }
+
+    /**
+     * Navigates on page with id.
+     *
+     * @param keyUrl - Url key.
+     * @param id     - Project id.
+     */
+    public static void navigatePageThroughId(final String keyUrl, final int id, final String endpoint) {
```

Comment:
This method is the same as the method called: navigatePageById

Reply:
done